samples/app-identity-linking-with-sso/csharp/IdentityLinkingWithSSO/Pages/tab.cshtml

```diff
@@ -181,29 +179,27 @@
                 }
             });
         }
-
         // Get client side token.
         function getClientSideToken() {
-
             return new Promise((resolve, reject) => {
             microsoftTeams.authentication.getAuthToken({
                 successCallback: (result) => {
                     resolve(result);
                 },
                 failureCallback: function (error) {
+                    console.log("id error");
                     reject("Error getting token: " + error);
                 }
             });
          });
         }
-
         // Get server side token and user profile.
         function getServerSideToken(clientSideToken) {
             return new Promise((resolve, reject) => {
                 microsoftTeams.getContext((context) => {
                     $.ajax({
                         type: 'POST',
-                        url: '/GetUserAccessToken',
+                        url: '/GetProfileOnBehalfOf',
```

Comment:
What profile we are getting here?
I think we are getting User details so can you please rename it accordingly.